modules/systemd/README.md

```diff
@@ -11,4 +11,23 @@ Those units are then copied into these folders (depending on unit base):
 `/usr/lib/systemd/system`  
 `/usr/lib/systemd/user`
 
-Supported management operations are [enabling](https://www.freedesktop.org/software/systemd/man/latest/systemctl.html#enable%20UNIT%E2%80%A6), [disabling](https://www.freedesktop.org/software/systemd/man/latest/systemctl.html#disable%20UNIT%E2%80%A6), [masking](https://www.freedesktop.org/software/systemd/man/latest/systemctl.html#mask%20UNIT%E2%80%A6%E2%80%A6) and [unmasking](https://www.freedesktop.org/software/systemd/man/latest/systemctl.html#unmask%20UNIT%E2%80%A6).
+Specific systemd config can be included in these directories, depending on certain usecases:  
+`config/systemd/system.conf.d`  
```

Comment:
These file paths to folders should have a trailing slash `config/systemd/system.conf.d/` etc.